pkg/openservicebroker/cmd/server/start.go

```diff
@@ -152,3 +185,15 @@ func (o TemplateServiceBrokerServerOptions) RunTemplateServiceBrokerServer(stopC
 	}
 	return server.GenericAPIServer.PrepareRun().Run(stopCh)
 }
+
+// these are used to set up for reading the config
+var (
+	groupFactoryRegistry = make(announced.APIGroupFactoryRegistry)
+	registry             = registered.NewOrDie("")
+	configScheme         = runtime.NewScheme()
+	configCodecs         = serializer.NewCodecFactory(configScheme)
+)
+
+func init() {
+	configinstall.Install(groupFactoryRegistry, registry, configScheme)
```

Comment:
> do we actually have to install/register the config resource? We don't anticipate anyone actually "creating"(or retrieving) a TSBConfig resource object, do we?

The installation provides a mechanism for controlled registration into the scheme (with all the error checking that provides).  It isn't directly related to a rest endpoint (note that all these are local).  It is technically possible for me to avoid it, but I doubt that anyone beyond myself and stefan could wire it.  I would suggest using the "normal" registration.